src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -60,14 +60,64 @@
 public class HeadlessGameServer {
 
   private static final Logger logger = Logger.getLogger(HeadlessGameServer.class.getName());
-  private static HeadlessGameServer instance = null;
   private final AvailableGames availableGames;
   private final GameSelectorModel gameSelectorModel;
-  private SetupPanelModel setupPanelModel = null;
   private final ScheduledExecutorService lobbyWatcherResetupThread = Executors.newScheduledThreadPool(1);
+  private final String startDate = TimeManager.getFullUtcString(Instant.now());
+  private static HeadlessGameServer instance = null;
+  private SetupPanelModel setupPanelModel = null;
   private ServerGame game = null;
   private boolean shutDown = false;
-  private final String startDate = TimeManager.getFullUtcString(Instant.now());
+
+  private HeadlessGameServer() {
+    super();
```

Comment:
This call to `super()` is redundant. HeadlessGameServer extends Object

Reply:
just movign this method as well..

Reply:
i did go ahead and took care of this: 12d4ce6

For code that is just moved, I want to avoid further changes or else the diff is huge and we need to start examining a lot more of the existing code.